src/processing/resources/index.js

```diff
@@ -9,6 +9,9 @@ import { ensureTrailingSlash } from '../../utils/url';
 
 function getResourceUrlReplacer (ctx) {
     return function (resourceUrl, resourceType, charsetAttrValue, baseUrl) {
+        if (!urlUtil.isSupportedProtocol(resourceUrl))
```

Comment:
So what was the cause of the problems and how what fixes it? Also missing regression tests